README.md

```diff
@@ -375,6 +375,9 @@ smtpd_sender_restrictions =
     reject_unknown_sender_domain,
     permit
 
+body_checks = pcre:/etc/postfix/body_checks
```

Comment:
I think this setup could be extracted in a separate doc to reduce the number of steps necessary to have SimpleLogin running.

Reply:
If we omit this from the README, I think ENFORCE_SPF should default to false in example.env, right?